src/main/resources/logback.xml

```diff
@@ -56,9 +56,9 @@
         <appender-ref ref="METRICS" />
     </root>
 
-    <logger name="io.iohk.ethereum.network.rlpx.RLPxConnectionHandler" level="INFO" />
+    <logger name="io.iohk.ethereum.network.rlpx.RLPxConnectionHandler" level="DEBUG" />
     <logger name="io.iohk.ethereum.blockchain.sync.SyncController" level="INFO" />
-    <logger name="io.iohk.ethereum.network.PeerActor" level="INFO" />
+    <logger name="io.iohk.ethereum.network.PeerActor" level="DEBUG" />
```

Comment:
Should we enable this permanently and remove this lines? (as we are setting the DEBUG option above)

Reply:
I think it should be a temporary change only. When we will have more peers we will be flooded by logs from there

Reply:
I'm not sure how temporal this will be :thinking: we still have this sort of logging on the other project